charts/latest/csi-driver-smb/templates/csi-smb-driver.yaml

```diff
@@ -3,6 +3,8 @@ apiVersion: storage.k8s.io/v1beta1
 kind: CSIDriver
 metadata:
   name: {{ .Values.driver.name }}
+parameters:
+  createSubDir: "true"
```

Comment:
let's only set `createSubDir` in storage class, and by default it's `true`

Reply:
fixed.